docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -138,11 +141,11 @@ Preparing For a Release
 
       git commit -am "Updated NEWS"
 
-#. Ensure the notes in `docs/advanced/whatsnew.rst <https://github.com/ClusterHQ/flocker/blob/master/docs/advanced/whatsnew.rst>`_ are up-to-date:
+#. Ensure the notes in `docs/releasenotes/index.rst <https://github.com/ClusterHQ/flocker/blob/master/docs/releasenotes/index.rst>`_ are up-to-date:
```

Comment:
Do we want this link here? It links to a copy of the release notes that is now unrelated to the release branch.  May be better to let the releaser find it in the branch.